Apps/APN-UIKit/APN UIKit/View/DashboardViewController.swift

```diff
@@ -33,21 +33,19 @@ class DashboardViewController: BaseViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
 
-        // In a future PR, we will remove the asyncAfter(). this is only for testing in sample apps because when app opens, the local queue is empty. so wait to check messages until first fetch is done.
-        DispatchQueue.main.asyncAfter(deadline: .now() + 5) { [self] in
-            inlineInAppViewCreatedInStoryboard.elementId = "dashboard-announcement"
-
-            // We want to test that Inline Views can be used by customers who prefer to use code to make the UI.
-            // Construct a new instance of the View, add it to the ViewController, then set constraints to make it visible.
-            let newInlineViewUsingUIAsCode = InAppMessageView(elementId: "dashboard-announcement-code")
-            // Because the Dashboard screen contains a lot of Views and it's designed using Storyboard, we are
-            // adding this inline View into the UI by adding to a StackView. This allows us to dyanamically add to the Dashboard screen without complexity or breaking any of the constraints set in Storyboard.
-            buttonStackView.addArrangedSubview(newInlineViewUsingUIAsCode)
-
-            // Customers are responsible for setting the width of the View.
-            newInlineViewUsingUIAsCode.translatesAutoresizingMaskIntoConstraints = false
-            newInlineViewUsingUIAsCode.widthAnchor.constraint(equalTo: buttonStackView.widthAnchor).isActive = true
-        }
+        // For inline Views added with Storyboard, set the elementId to finish setup of the View and begin showing messages.
```

Comment:
Just removed the `DispatchQueue.main.asyncAfter(deadline: .now() + 5) { [self]` and formatted the code. 

This delay is no longer needed after this commit since View will display messages that are fetched *after* the UI is in the foreground.